logback-core/src/main/java/ch/qos/logback/core/ConsoleAppender.java

```diff
@@ -77,27 +75,23 @@ private void targetWarn(String val) {
 
     @Override
     public void start() {
-        OutputStream targetStream = target.getStream();
         // enable jansi only if withJansi set to true
-        if (withJansi) {
-            targetStream = wrapWithJansi(targetStream);
-        }
-        setOutputStream(targetStream);
+        setOutputStream(withJansi ? getJansiStream() : target.getStream());
```

Comment:
Why change this?

Reply:
Indeed, I can revert. I found the 1 liner easier to read though.